go/webpack/cbor/cbor_reader.go

```diff
@@ -0,0 +1,71 @@
+package cbor
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+)
+
+// A Decoder helps callers iterate through a CBOR buffer, extracting item
+// headers and string bodies as they go.
+type Decoder struct {
+	full []byte
```

Comment:
Yeah, it's the full buffer that's being decoded. Does this just need a comment, or would you rather some other name?